src/test/java/com/ibanity/apis/client/products/xs2a/services/impl/SynchronizationServiceImplTest.java

```diff
@@ -51,18 +52,19 @@ void setUp() {
 
     @Test
     void create() throws Exception {
-        SynchronizationReadQuery synchronizationReadQuery =
-                SynchronizationReadQuery.builder()
+        SynchronizationCreationQuery synchronizationCreationQuery =
```

Comment:
If the customerOnline property is part of the response, I would suggest to update create_synchronization.json in order to return true aswell to make sure the property is properly mapped

Reply:
solved